app/views/admin/ofsted/_mini-table-child-services-register.html.erb

```diff
@@ -1,9 +1,13 @@
 <% if t.present? %>
+    <% showEndDateCol = t.find {|r| r.key?("EndDate") }.present? %>
     <table class="table">
         <thead>
             <th>Status</th>
             <th>Register</th>
             <th>Start date</th>
+            <% if showEndDateCol %>
```

Comment:
I find this kind of UI very confusing, because if the column isn't there it's hard to tell whether there's a bug or a regression, or if it's just that the data isn't there.

Could we just show the column all the time, but leave it blank when it's not set?